TESTS/netsocket/README.md

```diff
@@ -1068,527 +1088,513 @@ Test whether timeouts are obeyed in TCPSockets.
+   1. Record a time with millisecond precision.
+1. Repeat the test case 10 times.
 
 **Expected result:**
 
-Each send() call should return 100.
-
-Within each loop, one recv() may return the received packet size (100).
-Other calls should return `NSAPI_ERROR_WOULD_BLOCK`.
+Each `send()` call returns 100.
 
-When `NSAPI_ERROR_WOULD_BLOCK` is received, check that time consumed is
-more that 100 milliseconds but less than 200 milliseconds.
+Within each loop, one `recv()` may return the received packet size (100). Other calls return `NSAPI_ERROR_WOULD_BLOCK`.
 
+When `NSAPI_ERROR_WOULD_BLOCK` is received, check the time consumed is more that 100 milliseconds but less than 200 milliseconds.
 
 ### TCPSOCKET_SEND_TIMEOUT
 
 **Description:**
 
-Repeatedly send small packets in a given time limit
+Repeatedly send small packets in a given time limit.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
-3.  TCPSocket is open.
+1. Network interface and stack are initialized.
+1. Network connection is up.
+1. TCPSocket is open.
 
 **Test steps:**
 
-1.  Call `TCPSocket:connect("echo.mbedcloudtesting.com", 9);`.
-3.  Call `TCPSocket:set_blocking(false);`.
-3.  Call `TCPSocket:send("hello", 5);`.
-4.  Repeat 10 times.
-5.  Destroy the socket.
+1. Call `TCPSocket:connect("echo.mbedcloudtesting.com", 9);`.
+1. Call `TCPSocket:set_blocking(false);`.
+1. Call `TCPSocket:send("hello", 5);`.
+1. Repeat 10 times.
+1. Destroy the socket.
 
 **Expected result:**
 
-`TCPSocket::connect()` should return `NSAPI_ERROR_OK'.
-
-All send() calls should return in less than 800 milliseconds
+`TCPSocket::connect()` return `NSAPI_ERROR_OK`.
 
+All `send()` calls return in less than 800 milliseconds.
 
 ### TCPSOCKET_ENDPOINT_CLOSE
 
 **Description:**
 
-Test whether we tolerate endpoint closing the connection.
+Test whether you tolerate an endpoint closing the connection.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
-3.  TCPSocket is open.
+1. Network interface and stack are initialized.
+1. Network connection is up.
+1. TCPSocket is open.
 
 **Test steps:**
 
-1.  Call `TCPSocket::connect("echo.mbedcloudtesting.com", 13);`.
-2.  Call `TCPSocket::recv(<buffer>, 30);`.
-3.  Repeat until recv() returns 0.
-4.  Call `TCPSocket::close();`.
-5.  Delete socket.
+1. Call `TCPSocket::connect("echo.mbedcloudtesting.com", 13);`.
+1. Call `TCPSocket::recv(<buffer>, 30);`.
+1. Repeat until `recv()` returns 0.
+1. Call `TCPSocket::close();`.
+1. Delete the socket.
 
 **Expected result:**
-Connect should return `NSAPI_ERROR_OK`.
 
-First recv() should return more that zero. Something between 10 and 30 bytes (datetime string)
+Connect returns `NSAPI_ERROR_OK`.
 
-Second recv() should return zero because endpoint closed the connection.
-close() should return `NSAPI_ERROR_OK'.
+The first `recv()` returns more that zero. Something between 10 and 30 bytes (datetime string).
```

Comment:
What is this saying? That is returns more than 0? That it returns between 10 and 30? Both?

Reply:
I think both words are correct (means the same), but for consistency we can use "destroy" everywhere.

Reply:
Changed this sequence to:
The first `recv()` returns datetime string length (It is between 10 and 30 bytes)